cmake/Modules/FindClangFormat.cmake

```diff
@@ -0,0 +1,12 @@
+INCLUDE(FindPackageHandleStandardArgs)
+FIND_PROGRAM(CLANGFORMAT_EXECUTABLE clang-format HINTS ${CLANGFORMAT_ROOT})
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(clang-format DEFAULT_MSG CLANGFORMAT_EXECUTABLE)
```

Comment:
ok, i'll correct it in other files too

2015-08-22 19:45 GMT+03:00 Maxim Kot <notifications@github.com>:

> In cmake/Modules/FindClangFormat.cmake
> <https://github.com/libproject/libproject/pull/105#discussion_r37697310>:
>
> > @@ -0,0 +1,12 @@
> > +INCLUDE(FindPackageHandleStandardArgs)
> > +FIND_PROGRAM(CLANGFORMAT_EXECUTABLE clang-format HINTS ${CLANGFORMAT_ROOT})
> > +FIND_PACKAGE_HANDLE_STANDARD_ARGS(clang-format DEFAULT_MSG CLANGFORMAT_EXECUTABLE)
>
> Usualy find_package_handle_standard_args is putted at the end of script.
>
> It's common place where others will search it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/libproject/libproject/pull/105/files#r37697310>.
>



-- 
Respectfully,
Andrey Poltavets